AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Core/AWSAPICategoryPluginTests.swift

```diff
@@ -230,7 +230,7 @@ class AWSAPICategoryPluginTests: XCTestCase {
 
```

Comment:
please add inverted expectation on the publisher completion event in this test as well.